src/types/manifest.ts

```diff
@@ -0,0 +1,17 @@
+import { type Static, Type as T } from "@sinclair/typebox";
+import { StandardValidator } from "typebox-validators";
+
+export const commandSchema = T.Object({
+  description: T.String({ minLength: 1 }),
+  "ubiquity:example": T.String({ minLength: 1 }),
```

Comment:
why does example need a prefix?

Reply:
Not [manifest.json standard](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json). Check Mozilla developer network for it